nl/mail/chrome/messenger/updates/pbUpdates.properties

```diff
@@ -1 +1 @@
-UpToDate=Postbox %S is currently the newest version available.
+UpToDate=Postbox %S is op dit moment de nieuwste versie die beschikbaar is.
```

Comment:
@shermandickman This line is approved and doesn't need any corrections. The comment above refers to the whole file.